### ISSUE TITLE
Accessibility: Make Send and Attach File buttons real buttons

### ISSUE THREAD

**Issue by MarcoZehe** [threema-ch/threema-web#546]
## Expected Behavior

When using Threema Web with a screen reader, both items Send and Attach Files should be buttons. 

## Current Behavior

Currently, screen readers only announce them as clickable.

## Possible Solution

Either make them real `<button>` elements, or add `role="button" tabindex="0"` and a keyPress handler that does the same as the click handler and reacts to Space and Enter.

## Steps to Reproduce (for bugs)
1. Connect to ThreemaWeb with Firefox + NVDA (on Windows), or Safari or Chrome with VoiceOver turned on (on Mac).
2. Open a chat.
3. Find the Send and Attach Files button near the end of the output, after the input box.
4. Listen to what the screen reader says. In Firefox, you can also examine the Accessibility Tree with the new Accessibility Inspector, enabled in Dev Tools Settings.

## Your Environment
* Threema Web version: 2.0.0.8 Beta
* Threema App version: 4.0 Beta (iOS)
* Browser name and version: Firefox 61.0.1 on Windows
* Computer operating system and version: Windows 10 1803 (April 2018 update)
* Smartphone operating system and version: iOS 12 Public Beta 5, but also happens with 11.4.1.

**Comment by dbrgn** [comment]
Thanks for letting us know, @MarcoZehe, and for the detailed report! :slightly_smiling_face:

**Comment by SirTyson** [comment]
I can work on this.

**Comment by lgrahl** [comment]
@SirTyson That would be neat. :)
